src/renodeUtils.re

```diff
@@ -6,6 +6,47 @@ let charSep =
   | _ => '/'
   };
 
+let rec removeCommonPath = (~path1, ~path2) => {
```

Comment:
These switches can be replaced by ternary operators may be, but they are best at handling exceptions. Plz share if you have better suggestions about code here.

Reply:
I personally prefer switches, they make refactoring easier and more explicit the exceptions that are being handled. These look great!